Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -35,7 +33,8 @@ class XcodeSpec: QuickSpec {
 			_ = try? FileManager.default.removeItem(at: targetFolderURL)
 		}
 
-		describe("determingSwiftInformation") {
+		describe("determineSwiftInformation:") {
+			let currentSwiftVersion = swiftVersion.single()?.value
 			let testSwiftFramework = "Quick.framework"
 			let currentDirectory = URL(fileURLWithPath: FileManager.default.currentDirectoryPath, isDirectory: true)
```

Comment:
I think the current directory is guaranteed to be the directory that the test target is being run from. But I could be wrong on that.

Since the test is written as a quick spec, it is assumed that there will be a `Quick.framework` present. This assumption could be broken if Quick were integrated manually, integrated as a static framework, or if the current configuration were modified to load it from a different location.

In both cases, I think a failing test will give a good indication of where to sort out the problem. But perhaps a check to make sure that the test framework does in fact exist at the location it is assumed to be at would help someone disambiguate a test that failed because the test framework wasn't there versus one that failed because we couldn't determine it's Swift version. I went ahead and added that.

As far as writing a test that doesn't have these assumptions, I'm open to other suggestions on how to locate a framework that we are sure was compiled by the version of Swift used to compile the tests.